project/frontend/src/components/SwipeMatchSummary.js

```diff
@@ -0,0 +1,80 @@
+import './SwipeMatchSummary.css';
+
+import React from 'react';
+
+import { useHistory } from 'react-router-dom';
+
+import ConnectedUserChip from './ConnectedUserChip.js';
+import RestaurantCard from './RestaurantCard.js';
+
+import ThumbDown from '../assets/thumb_down.svg';
+import ThumbUp from '../assets/thumb_up.svg';
+
+function SwipeMatchSummary(props) {
+  const { restaurants, users, likes } = props;
+
+  const history = useHistory();
+
+  const topPick = restaurants[0];
```

Comment:
Will restaurants ever be empty?

Reply:
Yes it certainly will be empty before they are fetched from the Places API. In that case `topPick` will be undefined so the following if-else will make sure that no null pointers are hit (if `topPick` is undefined we just return `null` for the whole component which I don't think is an issue since this component is only every *visible* at the end of a session when restaurants are certainly populated).

Reply:
could we just have a return-early check for restaurants.length === 0 in that case? It'd be clearer to read that case
if (empty): return null;
return (<div...); 

(also would reduce the if-block level of nesting as a side effect :D)